include/sys/errno.h

```diff
@@ -176,8 +176,13 @@
 /* pseudo-errors returned inside kernel to modify return to process */
 #define EJUSTRETURN 256  /* don't modify regs, just return to userspace with */
                          /* current uctx (used by sigreturn and execve) */
-#define ERESTART 257     /* restart syscall */
 #define EPASSTHROUGH 258 /* ioctl not handled by this layer */
+/* These errors indicate when the current syscall should be restarted after
+ * being interrupted by a signal. Their semantics are the same as in Linux. */
+#define ERESTARTSYS                                                            \
+  259                      /* Restart iff no signal was caught                 \
```

Comment:
Could you do something with formatting here?
`#define ERESTARTSYS 259` is much more readable for me.

Reply:
Oh, I didn't notice it was formatted like this. `clang-format` is weird sometimes.